service/migration_manager.cc

```diff
@@ -883,7 +883,7 @@ future<std::vector<mutation>> prepare_new_view_announcement(storage_proxy& sp, v
         }
         mlogger.info("Create new view: {}", view);
         return seastar::async([&db, keyspace = std::move(keyspace), &sp, view = std::move(view), ts] {
-            auto mutations = db::schema_tables::make_create_view_mutations(keyspace, std::move(view), ts);
+            auto mutations = db::schema_tables::make_create_view_mutations(keyspace, view, ts);
             // We don't have a separate on_before_create_view() listener to
```

Comment:
I wonder what weird behavior we can get from this, when allocating tablets for views that happen in before_create_column_family(). Maybe we only use schema for unimportant stuff so that's why we haven't bumped into it before.

looks good to me.

/cc @nyh

Reply:
@tchaikov IIRC, 6.0 just branched out, so we might have to backport this fix to it, please double check.

it's great that we wired clang-tidy into our workflow, it's showing good fruits already :-)

Reply:
@raphaelsc thank you for point this out! i was under the wrong impression that branch-6.0 was not cut yet. and yes,  checked, we need to backport this change to 6.0 .

Reply:
I added this use-after-move in commit 88a5ddabcec625434b2df0ed82a63c36245af35f, which **did** work, and fixed a bug, so I would have known if it doesn't work (and worse, tries to dereference a null pointer). But we still need to fix the **potential** problem, so thanks.
I am guessing that the reason why no problem happened in simply that we have another copy of the shared pointer lying around (which makes sense, this is how shared pointers are usually used) which meant that the destructor didn't zero the pointer because it wasn't the last copy.